task_processing/metrics.py

```diff
@@ -0,0 +1,47 @@
+try:
+    import yelp_meteorite
+    METRICS_ENABLED = True
+except Exception:
+    METRICS_ENABLED = False
+
+
+_registered_metrics = {}
+
+
+def create_counter(name, dimensions={}):
+    if not METRICS_ENABLED:
+        return
+
+    if name not in _registered_metrics:
+        counter = yelp_meteorite.create_counter(
+            name, default_dimensions=dimensions)
+        _registered_metrics[name] = counter
+
+
+def create_timer(name, dimensions={}):
+    if not METRICS_ENABLED:
+        return
+
+    if name not in _registered_metrics:
+        timer = yelp_meteorite.create_timer(
+            name, default_dimensions=dimensions)
+        _registered_metrics[name] = timer
+
+
+def get_metric(name):
+    if METRICS_ENABLED:
+        return _registered_metrics.get(name)
+    else:
+        class DummyMetricType(object):
```

Comment:
Can we initialize this class in the module, create a global object and return the same object every time?